libraries/AP_HAL_ChibiOS/hwdef/luminousbee5/hwdef.dat

```diff
@@ -1,5 +1,8 @@
-# LUMINOUS BEE 5.1
+#42605 AND 42688 TOGETHER
+# Con i suggerimenti di andhi
+# LUMINOUS BEE 5.32/5.4 bidirectional telemetry - BETA 
 # hw definition file for processing by chibios_hwdef.py
+# aggiunto HAL_HAVE_FAST_MCU 1 
```

Comment:
This now should be
```
MCU_CLOCKRATE_MHZ 480
```

Reply:
I will report this to my colleague , thanks

Reply:
Resolved Now